webapp-django/crashstats/signature/views.py

```diff
@@ -35,36 +37,36 @@
 )
 
 
-def get_validated_params(request):
-    params = get_params(request)
-    if isinstance(params, http.HttpResponseBadRequest):
-        # There was an error in the form, let's return it.
-        return params
+def get_validated_params(view):
```

Comment:
"get_..." is a bad name. It's a decorator now. Consider "check_..." (or "pass_..." for somewhat consistency)